server/src/main/java/io/crate/analyze/SnapshotSettings.java

```diff
@@ -32,9 +32,14 @@ public final class SnapshotSettings {
 
     public static final Setting<Boolean> WAIT_FOR_COMPLETION = Setting.boolSetting("wait_for_completion", false);
 
+    // We add a postfix in order not break validation for partitioned tables
+    // whose indices must start with "partitioned" and thus cannot have custom prefix.
+    public static final Setting<String> POSTFIX = Setting.simpleString("postfix", "");
+
```

Comment:
We can use prefix add some extra handling into partitioned tables validation (ignore prefix before "partitioned") but I think it would be unnecessary increase of complexity. 

Main goal is to have a table with different name while table with old name is still on cluster, so postfix is fine (this options is also mentioned in the feature description)